app/client/jsx/Welcome.jsx

```diff
@@ -54,7 +54,7 @@ class Welcome extends Component {
         return (
             <div className="vestibule">
                 <h1>Welcome have fun</h1>
-                <input style={text_entry} className={name_opacity} type="text" placeholder="Name" name="name" minLength="1" onChange={this.handleDisplayNameChange.bind(this)}/><br/>
+                <input style={text_entry} autoComplete="off" className={name_opacity} type="text" placeholder="Name" name="name" minLength="1" onChange={this.handleDisplayNameChange.bind(this)}/><br/>
```

Comment:
I added this because it the autocomplete was slowing down my dev process on reload.